js/gratipay/team.js

```diff
@@ -192,6 +191,7 @@ Gratipay.team = (function() {
                     if(d.success) {
                         Gratipay.notification(d.success, 'success');
                     }
+                    debugger;
```

Comment:
Maybe they want you to declare an remote debugger or they'll simply ignore this instruction if none is provided? Can't find any documentation about this. I'll try by myself once @home :-)